src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -644,6 +647,31 @@ private bool ShouldRunBackupAfterTimerCallback(NextBackup backupInfo, out Period
             }
 
             var taskStatus = GetTaskStatus(topology, periodicBackup.Configuration);
+            if (_forTestingPurposes != null && _forTestingPurposes.SimulateClusterDownStatus)
+            {
+                taskStatus = TaskStatus.ClusterDown;
+                _forTestingPurposes.ClusterDownStatusSimulated = true;
+            }
+
+            string msg;
+            switch (taskStatus)
+            {
+                case TaskStatus.ActiveByCurrentNode:
+                    msg = $"Backup {backupInfo.TaskId}, current status is {taskStatus}, the backup will be executed on current node.";
+                    break;
+                case TaskStatus.ClusterDown:
+                    msg = $"Backup {backupInfo.TaskId}, current status is {taskStatus}, the backup will be rescheduled on current node.";
+                    var status = GetBackupStatus(backupInfo.TaskId, periodicBackup.BackupStatus);
+                    periodicBackup.UpdateTimer(GetTimer(periodicBackup.Configuration, status));
```

Comment:
if the cluster is down it will cause an infinite loop

Reply:
it will create a timer for next backup occurrence